### PR TITLE
Disable foundation tests since some are failing

### PR DESCRIPTION
**Pull request by blasten**
Log: https://api.cirrus-ci.com/v1/task/4826595086303232/logs/test.log

### CODE REVIEW

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: dev/bots/test.dart [hunk 1]
```diff
@@ -463,8 +463,8 @@ Future<void> _runTests() async {
 Future<void> _runWebTests() async {
   // TODO(yjbanov): re-enable when web test cirrus flakiness is resolved
   await _runFlutterWebTest(path.join(flutterRoot, 'packages', 'flutter'), tests: <String>[
-    'test/foundation/',
     // TODO(yjbanov): re-enable when flakiness is resolved
+    // 'test/foundation/',
```
Comment (zanderso) [inline dev/bots/test.dart]: Is there an issue filed for this? If so it should go in the TODO comment somewhere.

Comment (yjbanov) [reply dev/bots/test.dart]: It's https://github.com/flutter/flutter/issues/39991

**Comment by yjbanov** [comment]
Why was it necessary to disable these tests? Cirrus is configured to allow failures: https://github.com/flutter/flutter/blob/master/.cirrus.yml#L149

**Pull request merged**